src/equal.c

```diff
@@ -105,6 +111,10 @@ static int int_equal_scalar(const int* x, const int* y, bool na_equal) {
     return (xi == NA_INTEGER || yj == NA_INTEGER) ? NA_LOGICAL : xi == yj;
   }
 }
+static int raw_equal_scalar(const Rbyte* x, const Rbyte* y, bool na_equal) {
+  // raw vectors have no notion of missing value
```

Comment:
Comments should start with capital and no final stop, e.g. https://style.tidyverse.org/functions.html#comments-1